src/main/resources/database/changes/release-10.49.2/add-id-to-sampleunit.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE collectionexercise.sampleunit ADD COLUMN id UUID CONSTRAINT su_id_uuid UNIQUE;
```

Comment:
Is there no chance the same sample would be linked to multiple CE?

Reply:
Not sure how that would affect this - all we are doing is adding a UUID field to the sample unit that gets passed through from sample service

Reply:
Assuming we aren't going to link the exact same sample this should be ok